SConstruct

```diff
@@ -554,6 +554,13 @@ def config_env(toolchain, variant, env):
         if toolchain == 'clang':
             env.Append(CCFLAGS=['-Wno-redeclared-class-member'])
             env.Append(CPPDEFINES=['BOOST_ASIO_HAS_STD_ARRAY'])
+            try:
+                ldd_ver = subprocess.check_output([env['CLANG_CXX'], '-fuse-ld=lld', '-Wl,--version'],
+                                            stderr=subprocess.STDOUT).strip()
```

Comment:
do we want to check the contents of stdout (the version string) like in CMake or is zero exit status fine?